app/components/shared/main-menu.js

```diff
@@ -5,8 +5,21 @@ import { service } from '@ember/service';
 export default class SharedMainMenuComponent extends Component {
   @service currentSession;
   @service features;
+  @service impersonation;
 
   get showLegislatuurModule() {
     return this.currentSession.showLegislatuurModule;
   }
+
+  get areModulesShown() {
```

Comment:
why have this and now just showCommonModules?

Reply:
Yeah sometimes I think what if there are more.. but thats not the case atm so ill remove it